### ISSUE TITLE
training lose 1047081846814059528192.0000

### ISSUE THREAD

**Issue by Shaiken** [vijendra1125/Tensorfow_Object_detection_API-Custom_Mask_RCNN#32]
i take yout pic and training
when 
...
INFO:tensorflow:global step 1998: loss = 0.0773 (0.293 sec/step)
INFO:tensorflow:global step 1998: loss = 0.0773 (0.293 sec/step)
INFO:tensorflow:Saving checkpoint to path CP\model.ckpt
INFO:tensorflow:Saving checkpoint to path CP\model.ckpt
INFO:tensorflow:global step 1999: loss = 3.1521 (0.295 sec/step)
INFO:tensorflow:global step 1999: loss = 3.1521 (0.295 sec/step)
...
INFO:tensorflow:global step 2001: loss = 2.9904 (0.358 sec/step)
INFO:tensorflow:global step 2001: loss = 2.9904 (0.358 sec/step)
INFO:tensorflow:global step 2002: loss = 633318697598976.0000 (0.276 sec/step)
...

and check result test image1.jpg
use CP/model.ckpt-1999
no found anything 
![image](https://user-images.githubusercontent.com/30071132/85918153-c3ec4a00-b892-11ea-9928-d50e56a7f5af.png)

**Comment by vijendraxsingh** [comment]
Loss is too high at checkpoint 1999.


Best regards, 
Vijendra Singh

**Comment by Shaiken** [comment]
is that old version problem?
python                    3.5.6                he025d50_0
python-dateutil           2.8.0                    pypi_0    pypi
pytz                      2020.1                   pypi_0    pypi
pywin32                   228                      pypi_0    pypi
pywinpty                  0.5.7                    pypi_0    pypi
pyzmq                     19.0.1                   pypi_0    pypi
qtconsole                 4.7.5                    pypi_0    pypi
qtpy                      1.9.0                    pypi_0    pypi
send2trash                1.5.0                    pypi_0    pypi
setuptools                47.3.1                   pypi_0    pypi
six                       1.15.0                     py_0    anaconda
tensorflow-gpu            1.5.0                    pypi_0    pypi
tensorflow-tensorboard    1.5.1                    pypi_0    pypi

my chose model package
![image](https://user-images.githubusercontent.com/30071132/85944691-5eb85780-b96b-11ea-91d3-894e65a87d69.png)

**Comment by joelbudu** [comment]
@Shaiken found the solution.. check: https://github.com/vijendra1125/Custom-Mask-RCNN-Using-Tensorfow-Object-Detection-API/issues/25#issuecomment-699052106
